lms/views/api/lti.py

```diff
@@ -3,11 +3,13 @@
 
 from pyramid.view import view_config, view_defaults
 
-from lms.validation import APIRecordSubmissionSchema
+from lms.validation import APIRecordSpeedgraderSchema, APIRecordResultSchema
 from lms.services.lti_outcomes import LTIOutcomesRequestParams
 
 
-@view_defaults(request_method="POST", renderer="json")
+@view_defaults(
+    request_method="POST", renderer="json", permission="launch_lti_assignment"
```

Comment:
I slapped a permission here to keep the view from full-on crashing on a missing `Authorization` parameter/header. We need to do better, but this will prevent the thing from actually crashing for now.